blockchain/chain_test.go

```diff
@@ -15,6 +15,12 @@ import (
 	"github.com/gcash/bchutil"
 )
 
+const fixedExcessiveBlockSize uint32 = 64000000
```

Comment:
All these test invocations could be removed by putting in a default if the value is not set.

Reply:
I put this to make sure we test something besides the default EBS. I should probably set it to some odd value that will never be the actual default.

If I add a little documentation to the constant, would that be ok with you?

```go
// fixedExcessiveBlockSize should not be the default to ensure it will work in all cases
```

Reply:
That is super reasonable =)

Reply:
done. I think,